src/bugsnag.js

```diff
@@ -410,6 +410,63 @@
     enhance(history, "replaceState", wrapBuilder(buildReplaceState));
   }
 
+  function trackWorkers() {
+    if(!getBreadcrumbSetting("autoBreadcrumbsWorkers", true)) {
+      return;
+    }
+
+    var workers = 0;
+
+    window.Worker = hijackConstructor(window.Worker, function(url) {
```

Comment:
That actually makes a lot of sense, however as long as it's clear that's the case so you guys don't get a million reports of "some breadcrumbs aren't showing" 😜

That or documenting and recommending what you guys do internally as a solution? Kinda ugly, but much more open and less magic.